besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -588,24 +589,8 @@ void setBannedNodeIds(final List<String> values) {
   @Option(
       names = {"--pruning-enabled"},
       description =
-          "Enable pruning of world state of blocks older than the retention period (default: ${DEFAULT-VALUE})")
-  private final Boolean isPruningEnabled = false;
-
-  @Option(
-      names = {"--pruning-blocks-retained"},
-      hidden = true,
-      description =
-          "Minimum number of recent blocks for which to keep entire world state (default: ${DEFAULT-VALUE})",
-      arity = "1")
-  private final Long pruningBlocksRetained = DEFAULT_PRUNING_BLOCKS_RETAINED;
-
-  @Option(
-      names = {"--pruning-block-confirmations"},
-      hidden = true,
-      description =
-          "Minimum number of confirmations on a block before marking begins (default: ${DEFAULT-VALUE})",
-      arity = "1")
-  private final Long pruningBlockConfirmations = DEFAULT_PRUNING_BLOCK_CONFIRMATIONS;
+          "Enable pruning of world state of blocks older than the retention period (default: true if fast sync is enabled, false otherwise)")
```

Comment:
nit: Should we mention how many blocks is the default retention period?

Reply:
You're right that it's problematic that we mention the retention period without specifying what it is. As it's implementation specific and configured through `--X` flags, I might actually reduce the confusion by making it more high level. Thoughts?

Reply:
Maybe we can simplify the message and don't mention the retention period. @jakehaugen can help with the decision! :)